operator/bundle/manifests/platform.stackrox.io_securedclusters.yaml

```diff
@@ -372,6 +372,7 @@ spec:
                           in these section will not have any effect.
                         enum:
                         - EBPF
+                        - modern_bpf
```

Comment:
Why using snake case instead of upper camel case here?

Reply:
Using CORE_BPF now